src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const currentImg = document.querySelector('#largeImg');
+const thumbs = document.querySelector('#thumbs');
+
+thumbs.addEventListener('click', (bruh) => {
+  bruh.preventDefault();
+
+  if (bruh.target.tagName === 'IMG' || bruh.target.tagName === 'A') {
```

Comment:
The condition checks if the clicked element is an 'IMG' or 'A' tag. However, if the 'IMG' is clicked, 'bruh.target.closest('a')' might return null if the 'IMG' is not wrapped in an 'A' tag. Consider ensuring that 'bruh.target.closest('a')' is not null before accessing 'href'.